.circleci/config.yml

```diff
@@ -164,25 +154,11 @@ commands:
                 --fail \
                 --show-error \
                 "${EXTRA_HEADERS[@]}" \
-                https://api.github.com/repos/nomiclabs/hardhat/releases/latest \
-                | jq --raw-output .tag_name \
+                https://api.github.com/repos/nomiclabs/hardhat/releases \
+                | jq --raw-output 'map(select(.tag_name | test("^hardhat@"))) | .[0].tag_name' \
```

Comment:
Is the latest release always the first element in the list?

Reply:
It is according to my single, manual test :sweat_smile: Sadly, [the docs](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases) don't specify this.

But I think even in the (IMO unlikely) case that you get an older release, this is still better than failing every time the latest release is a Hardhat plugin, something that happens often enough.